packages/client/Atmosphere.ts

```diff
@@ -115,17 +123,31 @@ export default class Atmosphere extends Environment {
     })
   }
 
-  fetchHTTP = async (body: object, connectionId?: string) => {
-    const res = await fetch('/graphql', {
+  fetchHTTP = async (body: FetchHTTPData, connectionId?: string) => {
+    const req = {
       method: 'POST',
       headers: {
         accept: 'application/json',
         Authorization: this.authToken ? `Bearer ${this.authToken}` : '',
-        'content-type': 'application/json',
         'x-correlation-id': connectionId || ''
       },
-      body: JSON.stringify(body)
-    })
+    }
+    const uploadables = body.payload.variables?.uploadables
+    delete body.payload.variables?.uploadables
```

Comment:
hacky way around `graphql-trebuchet-client` not expecting `uploadables`. I forked and cloned the repo, but was unable to run `yarn build` successfully even with no changes to the code.

Reply:
well this is interesting! what was the error?

Reply:
I saw that you upgraded already but I was just getting a bunch of type errors previously. not sure if i was missing s step before building.

```
clean-webpack-plugin: /Users/tiffanyhan/resources/graphql-trebuchet-client/dist/**/* has been removed.
ℹ ｢atl｣: Using typescript@3.9.7 from typescript
ℹ ｢atl｣: Using tsconfig.json from /Users/tiffanyhan/resources/graphql-trebuchet-client/tsconfig.json
ℹ ｢atl｣: Checking started in a separate process...
✖ ｢atl｣: Checking finished with 7 errors
Hash: e7386da1a7fd1ad8d6fd
Version: webpack 4.16.1
Time: 2711ms
Built at: 11/17/2020 11:37:42 AM
                        Asset       Size  Chunks             Chunk Names
                     index.js   8.38 KiB   index  [emitted]  index
                 index.js.map   11.1 KiB   index  [emitted]  index
             types/index.d.ts  181 bytes          [emitted]  
     types/GQLHTTPClient.d.ts  491 bytes          [emitted]  
types/GQLTrebuchetClient.d.ts   1.89 KiB          [emitted]  
Entrypoint index = index.js index.js.map
[./src/GQLHTTPClient.ts] 553 bytes {index} [built]
[./src/GQLTrebuchetClient.ts] 2.52 KiB {index} [built]
[./src/index.ts] 146 bytes {index} [built]

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:76:18 
    TS2769: No overload matches this call.
  Overload 1 of 2, '(event: Events.DATA | Events.CLOSE | Events.TRANSPORT_SUPPORTED, listener: (m: string | boolean | object | ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView | ClosePayload, ...args: any[]) => void): any', gave the following error.
    Argument of type '"data"' is not assignable to parameter of type 'Events.DATA | Events.CLOSE | Events.TRANSPORT_SUPPORTED'.
  Overload 2 of 2, '(event: Events.KEEP_ALIVE | Events.TRANSPORT_CONNECTED | Events.TRANSPORT_RECONNECTED | Events.TRANSPORT_DISCONNECTED, listener: () => void): any', gave the following error.
    Argument of type '"data"' is not assignable to parameter of type 'Events.KEEP_ALIVE | Events.TRANSPORT_CONNECTED | Events.TRANSPORT_RECONNECTED | Events.TRANSPORT_DISCONNECTED'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:79:18 
    TS2769: No overload matches this call.
  Overload 1 of 2, '(event: Events.DATA | Events.CLOSE | Events.TRANSPORT_SUPPORTED, listener: (m: string | boolean | object | ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView | ClosePayload, ...args: any[]) => void): any', gave the following error.
    Argument of type '"reconnected"' is not assignable to parameter of type 'Events.DATA | Events.CLOSE | Events.TRANSPORT_SUPPORTED'.
  Overload 2 of 2, '(event: Events.KEEP_ALIVE | Events.TRANSPORT_CONNECTED | Events.TRANSPORT_RECONNECTED | Events.TRANSPORT_DISCONNECTED, listener: () => void): any', gave the following error.
    Argument of type '"reconnected"' is not assignable to parameter of type 'Events.KEEP_ALIVE | Events.TRANSPORT_CONNECTED | Events.TRANSPORT_RECONNECTED | Events.TRANSPORT_DISCONNECTED'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:82:11 
    TS2345: Argument of type '{ id: string; type: string; payload: OperationPayload; }' is not assignable to parameter of type 'Data'.
  Object literal may only specify known properties, and 'id' does not exist in type 'ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:121:28 
    TS2345: Argument of type '{ id: string; type: string; }' is not assignable to parameter of type 'Data'.
  Object literal may only specify known properties, and 'id' does not exist in type 'ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:140:28 
    TS2345: Argument of type '{ id: string; type: string; payload: OperationPayload; }' is not assignable to parameter of type 'Data'.
  Object literal may only specify known properties, and 'id' does not exist in type 'ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:142:43 
    TS2345: Argument of type '{ type: string; payload: OperationPayload; }' is not assignable to parameter of type 'Data'.
  Object literal may only specify known properties, and 'payload' does not exist in type 'ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.

ERROR in [at-loader] ./src/GQLTrebuchetClient.ts:153:26 
    TS2345: Argument of type '{ id: string; type: string; payload: OperationPayload; }' is not assignable to parameter of type 'Data'.
  Object literal may only specify known properties, and 'id' does not exist in type 'ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```